python/GafferUI/RandomUI.py

```diff
@@ -67,6 +73,7 @@
 			values are different.
 			""",
 
+			"nodule:type", "",
```

Comment:
Rather than do this for every plug, we'd probably be better to use a [wildcarded registration](https://github.com/GafferHQ/gaffer/blob/master/python/GafferSceneUI/SceneNodeUI.py#L55). But actually, if we don't derive AuxiliaryNodeGadget from StandardNodeGadget we don't have this issue anyway, because it won't have any mechanism for managing nodules at all.